server/index.js

```diff
@@ -32,6 +32,7 @@ mongoose.connect(dbUrl, { useNewUrlParser: true }, unlockLocks);
 mongoose.set('useCreateIndex', true);
 mongoose.set('useFindAndModify', false);
 
+app.enable('trust proxy');
```

Comment:
extract this string to some enum value 

```
const AppConfigs = Object.freeze({
     PROXY: 'trust proxy'
});
```

and use it like this

`app.enable(AppConfigs.PROXY);`